Readme.md

```diff
@@ -2,15 +2,16 @@
 
 ## Windows
 
-|Branch|Unit Tests|Functional Tests|
-|:--:|:--:|:--:|
-|**master**|[![Build status](https://gvfs.visualstudio.com/ci/_apis/build/status/CI%20-%20Windows%20-%20master?branchName=master)](https://gvfs.visualstudio.com/ci/_build/latest?definitionId=7)|[![Build status](https://gvfs.visualstudio.com/ci/_apis/build/status/CI%20-%20Windows%20-%20Full%20Functional%20Tests?branchName=master)](https://gvfs.visualstudio.com/ci/_build/latest?definitionId=6)|
-
+|Branch|Unit Tests|Functional Tests|Large Repo Perf|

```

Comment:
Is the Large Repo Build test also a CI test?

Reply:
No, but it is scheduled to run nightly so decided to add it.

Reply:
Sorry, what I meant was could we also add Large Repo Build test? (I'm not sure if it's run automatically as well)

Reply:
Sorry, I could add that but right now it is only ran on the PRs so rarely ran on master or shipped.  I could set up a CI build so that we would have that as well.  I could also setup the Large Repo Perf build to run as CI instead of nightly but running nightly makes it easy to find the last run against master to compare performance.

Reply:
I think the other advantage to nightly is reducing the contention for the one machine that can run the tests.  I don't think you need to change the Build test settings as part of this PR, I was thinking if it was already running nightly though we might as well add a badge for it.

Reply:
I added a scheduled nightly build for the Large Repo Build and going to add those build badges.